File: C#/RefactoringWorkshop/RefactoringWorkshop/Coupler/MessageChain/ArticleWrapper.cs
using System.Collections.Generic;

namespace RefactoringWorkshop.Coupler.MessageChain
{
    public class ArticleWrapper
    {
        public List<string> Companies = new List<string>();
        public List<string> EntityCodes = new List<string>();
        public string ArticleRef;

        public void AddCompanies(string company) {
            Companies.Add(company);
        }

        public void AddEntityCodes(string code) {
            EntityCodes.Add(code);
        }

    }
}

File: Java/RefactoringWorkshop/src/main/java/coupler/messagechain/ArticleWrapper.java
package coupler.messagechain;

import java.util.ArrayList;
import java.util.List;

public class ArticleWrapper {
    private List<String> companies = new ArrayList<>();
    private List<String> entityCodes = new ArrayList<>();
    private String articleRef;

    public void setArticleRef(String articleRef) {
        this.articleRef = articleRef;
    }

    public void addCompanies(String company) {
        this.companies.add(company);
    }

    public void addEntityCodes(String code) {
        this.entityCodes.add(code);
    }
    public List<String> getEntityCodes() {
        return this.entityCodes;
    }

    public List<String> getCompanies() {
        return this.companies;
    }

    public String getArticleRef() {
        return articleRef;
    }
}
